package.json

```
@@ -41,23 +46,30 @@
     "gts": "^3.1.0",
     "husky": "^8.0.1",
     "jest": "^29.0.3",
-    "jest-environment-jsdom": "^29.0.3",
+    "jest-environment-jsdom": "^29.3.1",
     "jest-junit": "^14.0.1",
+    "jsdom": "^20.0.3",
     "lint-staged": "^13.0.3",
+    "live-server": "^1.2.2",
+    "mini-css-extract-plugin": "^2.7.0",
     "prettier": "^2.7.1",
-    "ts-jest": "^29.0.1",
+    "ts-jest": "^29.0.3",
+    "ts-loader": "^9.4.1",
     "ts-node": "^10.4.0",
     "tsconfig-paths": "^4.1.0",
     "typescript": "^4.0.3",
-    "validate-branch-name": "^1.3.0"
+    "validate-branch-name": "^1.3.0",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.0",
+    "webpack-merge": "^5.8.0"
   },
   "lint-staged": {
     "*.{js,jsx,ts,tsx}": "eslint --fix",
     "*.{json,scss}": "prettier --write",
     "../.circleci/config.yml": "make validate-circleci"
   },
   "validate-branch-name": {
-    "pattern": "^main$|^(TT|tt)()(-\\d+)(-[a-z-]+)?$",
+    "pattern": "^main$|^(IN|in)()(-\\d+)(-[a-z-]+)?$",
```

```suggestion
    "pattern": "^main$|^(TT|IN|DEV)-\\d+(-[a-z0-9-]+)?$",
```
This allows tickets in any project to create branches and make changes without having to keep changing this in every project. Also I'd like to keep some consistency here rather than each individual project having different regex. Happy to make changes to it, but I'd prefer it go through ADR, agree by the team and rolled out everywhere including the boilerplate.